src/action/views/run_canvas_email.py

```diff
@@ -76,6 +76,8 @@ def run_canvas_email_action(
 
             return redirect('action:item_filter')
 
+        set_action_payload(req.session, action_info.get_store())
```

Comment:
This fix is partial and in fact points to a different error in the code. I'll address the error in development instead and reject this merge